src/main/java/com/synopsys/integration/alert/web/controller/UIComponentController.java

```diff
@@ -72,15 +72,16 @@ public UIComponentController(final DescriptorMap descriptorMap) {
                 // filter by type also
                 if (StringUtils.isNotBlank(context)) {
                     final ConfigContextEnum contextType = EnumUtils.getEnum(ConfigContextEnum.class, context);
-                    final UIConfig uiConfig = descriptor.getUIConfig(contextType);
-                    if (uiConfig != null) {
-                        return Arrays.asList(uiConfig.generateDescriptorMetadata());
+                    final Optional<DescriptorMetadata> uiConfig = descriptor.getMetaData(contextType);
+                    if (uiConfig.isPresent()) {
+                        return Arrays.asList(uiConfig.get());
                     } else {
                         return Collections.emptyList();
```

Comment:
I know you didn't add these, but we might as well convert these to `List.of()` as well.